pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: file
-version: 3.0.0
+version: 3.0.1
```

Comment:
`4.0.0` since the sdk constraints changed from Dart 1 to Dart 2

Reply:
done.